runtime/src/bank.rs

```diff
@@ -323,12 +326,13 @@ mod executor_cache {
     }
 }
 
-const MAX_CACHED_EXECUTORS: usize = 100; // 10 MB assuming programs are around 100k
+const MAX_CACHED_EXECUTORS: usize = 256;
```

Comment:
Let's assume that each child bank evicts 10% of the cache (I sampled 2000 slots and found an average of 8 executor insertions per block) and so that means we cache approx 10 additional executors per unrooted bank. If we assume that validators manage about 35 unrooted banks, that's 350 additional executors in memory beyond the root executors cache size of 100. Then assume that programs are about 1MB in size (not 100KB) and that JIT compiled programs get paged. That brings us to (100 + 35 * 10) * 1MB ~= half a gig of RAM.

My hope is that with a larger executors cache, we see much less eviction in child banks. If we conservatively estimate that we get down to 5 evictions per block, then we'll have (256 + 35 * 5) * 1MB ~= half a gig of RAM.